[dash] Initial port of prev/next subnav, added to top & bottom on page

chalin opened the pull request:
For Shams! With this initial port, you need to provide the `prev` and `next` links.

Fixes #1322

Staged, e.g., see: https://ng2-io.firebaseapp.com/get-started/install

cc @fertolg 

--- 

This is what it looks like:

<img width="333" alt="screen shot 2018-10-01 at 13 35 18" src="https://user-images.githubusercontent.com/4140793/46305915-d253e780-c580-11e8-9604-8afe66abf06e.png">

Notice how long titles are shortened on narrow displays:

<img width="344" alt="screen shot 2018-10-01 at 13 34 56" src="https://user-images.githubusercontent.com/4140793/46305950-e1d33080-c580-11e8-8f9d-730f64713c14.png">

fertolg commented on src/_includes/subnav.html:
```diff
@@ -0,0 +1,16 @@
+{% if page.next or page.prev -%}
+<nav id="subnav">
```
this will output two navs with the same ID which is not good practice, we may want to change this to a class?

chalin replied:
Yes, thanks! Done

fertolg commented on src/_includes/subnav.html:
```diff
@@ -0,0 +1,16 @@
+<nav id="subnav">
+  <ul>
+    <li class="previous">
+      {% if page.prev.path -%}
```
can we move the if statement to wrap the <li> so that it won't output anything when there's no previous or next?

chalin replied:
I've fixed the styling issues and now I can move the `<li>` inside the if.

fertolg commented on src/_assets/css/_subnav.scss:
```diff
@@ -0,0 +1,63 @@
+// in particular, adjust dimensions (that are currently in px).
+//
+
+#subnav {
```
we should add `clear: both` to the subnav so that it doesn't overlap with floating images like in step 4.

chalin replied:
Done

Sfshaza commented:
Magnifique!!!  (What's LGTM in French?)

chalin commented:
> Magnifique!!! (What's LGTM in French?)

Magnifique is much more poetic! :)

chalin commented on src/get-started/test-drive.md:
```diff
@@ -186,5 +186,3 @@ You should see the updated string in the running app almost immediately.
 
 Let's learn some core Flutter concepts, by creating a small app.
-
-[Next step: Write your first app](/get-started/codelab)
```
@Sfshaza FYI, I dropped the hardcoded "Next step" links (since each Get Started page now has next/prev links at the top and bottom of the pages.

chalin commented:
@fertolg - I reworked the styles and removed the link icon when no next/prev link is present. Travis is happy so I'll merge now, but comments on the latest changes are still welcome. Thanks.
